lua/lspconfig/idris2.lua

```diff
@@ -0,0 +1,32 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.idris2 = {
+  default_config = {
+    cmd = { 'idris2-lsp' },
+    filetypes = { 'idris2' },
+    root_dir = util.root_pattern '*.ipkg',
+  },
+  docs = {
+    description = [[
+https://github.com/idris-community/idris2-lsp
+
+Idris 2 language server.
+
+Note that this requires a build of Idris 2 that includes the Idris 2 API
+package. For instructions, see
+<https://github.com/idris-lang/Idris2/blob/main/INSTALL.md>. Steps 5 and 8 are
```

Comment:
Can you use standard markdown link formatting here?

Reply:
I re-worded this section. But note that Github does recognize `<>` for "link literals", e.g. `<https://example.net>` renders as a hyperlink: <https://example.net>, not sure if that was the concern.